### PR TITLE
aruha-1322: changelog.md

### PR DESCRIPTION
**Pull request by adyach**
aruha-1322: changelog.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.3.0] - 2017-11-14
+
 ### Added
+- Switch from local token info service to remote one
```
Comment (rcillo) [inline CHANGELOG.md]: @adyach I think it's the other way around "switch from remote token info to local token info service"

Comment (adyach) [reply CHANGELOG.md]: but the default one is local, so you can switch to remote by turning the feature on

Comment (rcillo) [reply CHANGELOG.md]: ok, got it

**Comment by adyach** [comment]
👍

**Pull request merged**